### ISSUE TITLE
Encoding Settings window on top of main window (5.3 beta et al) + a suggestion

### ISSUE THREAD

**Issue by iamjaydear** [RandomEngy/VidCoder#498]
It would be good if the encoding settings window would not be kept "on top" of the main window when the main window is bigger and selected, and vice versa, like normal Windows do.  Suggestion: I suppose you may have come across Imgburn in your travels. It has a cool feature where it's main window and log window sort of snap to the same width and stick loosely to each other so that you don't have to keep moving two separate windows around and reorganise them repeatedly. 'Twould be nice if you could emulate this behaviour in VC. There may be others like me that keep the main and encoding settings windows open all the time.

**Comment by RandomEngy** [comment]
I agree. Adding some kind of docking system would be ideal. It's been something I've been investigating for a while. I looked at AvalonDock but it doesn't do what I would need it to do. I also looked into some paid options but they are not compatible with open source software. My best lead right now would be to adapt the docking code from Windows Performance Analyzer.

I haven't found the time to pursue that lead more; it would likely require pulling out a bunch of code and convincing another team to set up a new library.

Or building my own docking library from scratch, which I don't have the time to do. But yes, it is an eventual aspiration.
